### DISCUSSION TITLE
Issue on websocket connection

### DISCUSSION THREAD

**Discussion by Karthigasrini** [blockscout/blockscout#9850] [Q&A]
Hi,
 Blockscout frontend running fine, but the websocket URL returns error. Without refreshing a site, Blocks are not updating automatically.
 Can you provide me solution?
 
 **phoenix.mjs:867 WebSocket connection to 'wss://testnet.example.net/socket/v2/websocket?vsn=2.0.0' failed:** 
 
﻿

**Comment by ekawibisono** [comment]
Hi @Karthigasrini Your problem has solved ? or not ?
